plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestKafkaWithConfluentSchemaRegistryMinimalFunctionality.java

```diff
@@ -73,6 +73,11 @@
             .name("col_3").type().optional().doubleType()
             .endRecord();
 
+    private TestingKafka createWithSchemaRegistry()
```

Comment:
nit : Can we capture this change in a separate commit.. Not mandatory

Reply:
Will look into why this works and update.